src/main/java/org/folio/inventory/consortium/handlers/MarcInstanceSharingHandlerImpl.java

```diff
@@ -74,6 +93,63 @@ public Future<String> publishInstance(Instance instance, SharingInstance sharing
       });
   }
 
+  private Future<Record> unlinkAuthorityLinksIfNeeded(Record marcRecord, String instanceId, Context context, Storage storage) {
```

Comment:
should we change the name to `detachLocalAuthorityLinksIfNeeded` since there isn't a javadoc to explain further?

Reply:
Renamed, thanks